mathesar/api/utils.py

```diff
@@ -59,3 +60,41 @@ def _get_record_dict(record):
         output_groups = None
 
     return processed_records, output_groups
+
+
+def is_number(column_type):
+    """
+    Check if a column data type is a number
+    Args:
+        column_type: data type of column
+    """
+    return str(column_type) in [
+        'INTEGER',
+        'BIGINT',
+        'SMALLINT',
+        'NUMERIC',
+        'REAL',
+        'DOUBLE_PRECISION'
+    ]
+
+
+def follows_json_spec(number):
```

Comment:
Please rename this to make it obvious that we're asserting whether a string conforms to the JSON number definition.

Reply:
How about `follows_json_number_spec`?

Reply:
Sure, that works.